### DISCUSSION TITLE
Failed to look up the in-house Terraform module's version tag in Enterprise github

### DISCUSSION THREAD

**Discussion by jiwoorico** [renovatebot/renovate#22667] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.115.1

### If you're self-hosting Renovate, select which platform you are using.
GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'd like the Renovate could find the terraform module's newer tag version in another repo.

### What you tried so far.

Here's my Terraform script main.tf in a repo under the **org-name**. **tf-modules-abc** is a Terraform module repo which includes a few tags, like v0.2.1,  v0.2.2, v0.2.3.

```
module "vpc" {
  source  = "terraform-aws-modules/vpc/aws"
  version = "3.14.0"
}



module "abc" {
  source = "git::git@git.company.com:org-name/tf-modules-abc.git//?ref=v0.2.1"
}
```

following is the renovate.json:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "terraform": {
    "enabled": true
  },
  "packageRules": [
    {
      "matchDatasources": ["terraform-module"],
      "matchPackageNames": ["*abc*"],
      "automerge": false
    }
  ]
}
```

The current result is that Renovate could find the Terraform vpc module's newer version, but regarding the abc module, the Renovate reported following errors in the Dependency Dashboard:

_**Dependency Lookup Warnings ⚠
Renovate failed to look up the following dependencies: Failed to look up git-tags package git@git.company.com:org-name/tf-modules-abc.git//.
Files affected: main.tf**_ 

Could someone help take a look whether there's any config I missed?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by jiwoorico** [comment]
And in the Detected Dependencies under the Dependancy Dashboard, it seems Renovate has successfully detected the current tag version:

```
terraform
main.tf
git.company.com:org-name/tf-modules-abc.git v0.2.1
terraform-aws-modules/vpc/aws 3.14.0
```

**Comment by jiwoorico** [comment]
here're the test repos for reproduce:

TF module: https://github.com/jiwoorico/tf-module-ec2
repo to consume the TF module: https://github.com/jiwoorico/tf-ec2-test

**Reply by rarkins** [reply]
Do you mean by this that it reproduces on GitHub.com?

**Reply by jiwoorico** [reply]
> Do you mean by this that it reproduces on GitHub.com?

yes, but I'm not sure whether I didn't config the renovate correctly.

**Reply by jiwoorico** [reply]
I could see the error msg on this page:

https://github.com/jiwoorico/tf-ec2-test/issues/4

**Reply by rarkins** [reply]
Can you check the logs? Accessible via https://developer.mend.io

**Reply by jiwoorico** [reply, marked as answer]
ok, that's a great tool. I found there was a typo in the repo name. After I fixed the typo, the terraform module tag version works. Thanks.
